src/main/java/org/opensrp/web/rest/OrganizationResource.java

```diff
@@ -185,5 +189,28 @@ public void setPractitionerService(PractitionerService practitionerService) {
 			return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
 		}
 	}
+	
+	@RequestMapping(value = "/search", method = RequestMethod.GET, produces = { MediaType.APPLICATION_JSON_VALUE })
+	public ResponseEntity<String> searchOrganization(OrganizationSearchBean organizationSearchBean)
+	    throws JsonProcessingException {
+		
+		OrganizationSearchcBean response = new OrganizationSearchcBean();
```

Comment:
Correct typo in name of object

Reply:
rename OrganizationSearchcBean to OrganizationResponseBean